### PR TITLE
Added the first post endpoint for adding comments to articles

### PR DESCRIPTION
**Pull request by Karumu05**

### CODE REVIEW

File: __tests__/mainTestSuite.test.js [hunk 1]
```diff
@@ -169,3 +169,70 @@ describe('GET /api/articles/:article_id/comments', () => {
...
+  it('should respond with an appropriate error message when given a valid but non-existent id', () => {
+    return request(app)
+    .post('/api/articles/99999/comments')
+    .send({
+      username: 'butter_bridge',
+      body: 'Commenting, commenting everyone loves to comment'
+    })
+    .expect(404)
+    .then((result) => {
+      expect(result.body.msg).toBe('Invalid inputted data')
```
Comment (ghost) [inline __tests__/mainTestSuite.test.js:200]: I'd consider this error message, how useful is it? The input is totally valid, so is the id. It just doesn't exist.

File: __tests__/mainTestSuite.test.js [hunk 2]
```diff
@@ -169,3 +169,70 @@ describe('GET /api/articles/:article_id/comments', () => {
...
+    return request(app)
+    .post('/api/articles/2/comments')
+    .send({
+      username: 'IAMaCommentor',
+      body: 'Commenting, commenting everyone loves to comment'
+    })
+    .expect(404)
+    .then((result) => {
+      expect(result.body.msg).toBe('Invalid inputted data')
+    })
```
Comment (ghost) [inline __tests__/mainTestSuite.test.js:236]: Nice work. You could consider some extra edge cases of extra keys

**Review by ghost — COMMENTED** [review]
Nice work on this one, some edge cases to consider but no need to re submit.
